test/e2e/helpers.go

```diff
@@ -57,7 +57,7 @@ import (
 
 const (
 	labelControlPlaneNode = "node-role.kubernetes.io/control-plane"
-	prowjobImage          = "quay.io/kubermatic/build:go-1.23-node-20-6"
+	prowjobImage          = "quay.io/kubermatic/build:go-1.24-node-20-0"
```

Comment:
Go update requires a bit more work, leaving a comment so we don't forget about that (there should probably be an issue for it).